.github/workflows/cd-cloud-dev.yml

```diff
@@ -17,4 +19,4 @@ jobs:
           service_account_key: '${{ secrets.DEV_GCP_SA_KEY }}'
 
       - name: Deploy to dev GAE project
-        run: 'gcloud app deploy app.yaml --quiet --no-promote --version v1'
+        run: 'gcloud app deploy app.yaml --quiet --no-promote --version ${{ env.TAG }}'
```

Comment:
should we drop `--no-promote`?

Reply:
yeah while we're at it, dropping quiet as well since i'd like more noise to see if things go wrong